homework-submission/week2/src/App.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import Description from './ToDo/Description';
+import DeadLines from './ToDo/DeadLine';
+import DoneCheckBox from './ToDo/DoneCheckBox';
+//import DoneCheckBox from './ToDo/DoneCheckBox';
+
+let isChecked = false;
+
+const todos = [
+  {
+    "id": 1,
+    "description": "Get out of bed",
+    "deadline": "2017-09-11",
+    "done": true
+  },
+  {
+    "id": 2,
+    "description": "Brush teeth",
+    "deadline": "2017-09-10",
+    "done": false
+  },
+  {
+    "id": 3,
+    "description": "Eat breakfast",
+    "deadline": "2017-09-09",
+    "done": false
+  }
+]
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { isChecked: this.props.done };
+    
```

Comment:
this.state = { todos : [ ... ] }

Reply:
Hi Jalal. can you explain 'this.state = { todos : [ ... ] }' more? I don't understand how should I write.

Reply:
this.state = { todos : [ ... this.props.done] }